lib/wsdl.js

```diff
@@ -1774,7 +1774,7 @@ WSDL.prototype.objectToXML = function(obj, name, nsPrefix, nsURI, isFirst, xmlns
     }
   } else if (typeof obj === 'object') {
     for (name in obj) {
-      if (!obj.hasOwnProperty(name)) continue;
+      if (!({}).hasOwnProperty.call(obj, name)) continue;
```

Comment:
Please, refer to it as `Object.hasOwnProperty`, so that it's more readable what is going on here, thanks a lot 👍